doc/en/conf.py

```diff
@@ -141,10 +141,18 @@
 linkcheck_ignore = [
     "https://blogs.msdn.microsoft.com/bharry/2017/06/28/testing-in-a-cloud-delivery-cadence/",
     "http://pythontesting.net/framework/pytest-introduction/",
-    r"https://github.com/pytest-dev/pytest/issues/\d+",
-    r"https://github.com/pytest-dev/pytest/pull/\d+",
+    r"https://pypi\.org/project/pytest.*",
+    r"https://github\.com/sponsors/.*",
```

Comment:
Why is this necessary? I don't remember such URLs not working in other projects.

Reply:
```
    r"https://github.com/pytest-dev/pytest/issues/\d+",
    r"https://github.com/pytest-dev/pytest/pull/\d+",
```
this is from the original version. But you're right, this doesn't trigger any issues with link.

Reply:
Without `r"https://github\.com/sponsors/.*",` checking will trigger issues:  
WARNING: broken link: https://github.com/sponsors/xxx (429 Client Error: Too Many Requests for url: https://github.com/xxx)

It seems to be related to missing credentials or tokens for GitHub but I'm not 100% sure.

Reply:
With `xxx` specifically, I suppose? How about with real existing usernames?